io.catenax.sealant.sealant_pass/1.0.0/SealantPass.ttl

```diff
@@ -0,0 +1,648 @@
+    bamm:preferredName "supplier entity"@en;
+    bamm:description "Entity encapsulating the information from the supplier."@en.
+:HazardsIdentificationEntity a bamm:Entity;
+    bamm:properties (:classificationGHS :possibleHazards :hazardousIngredients);
+    bamm:preferredName "Hazards identifiaction entity"@en;
+    bamm:description "Entity encapsulating the hazards on a substance or mixture."@en.
+:ProductCompositionEntity a bamm:Entity;
+    bamm:properties (:substance :mixture);
+    bamm:preferredName "Product composition entity"@en;
+    bamm:description "Entity encapsulating the product composition informations."@en.
+:SafetyMeasureEntity a bamm:Entity;
+    bamm:properties (:firstAid :fireExtinguisher :personalPrecaution :technicalMeasures :exposureLimit);
+    bamm:preferredName "Safety measure entity"@en;
+    bamm:description "Entity encapsulating the safety measures."@en.
+:ProductPropertyEntity a bamm:Entity;
+    bamm:properties (:physicalState :colour [
+  bamm:property :odour;
+  bamm:optional "true"^^xsd:boolean
+] :pH :phaseTransition [
+  bamm:property :flashPoint;
+  bamm:optional "true"^^xsd:boolean
+] :density :reactivity);
+    bamm:preferredName "product property entity"@en;
+    bamm:description "Entity encapsulating the product properties."@en.
+:ProductDisposalEntity a bamm:Entity;
+    bamm:properties (:wasteManagement :wasteInformation);
+    bamm:preferredName "Product disposal entity"@en;
+    bamm:description "Entity encapsulating the information needed for the product disposal."@en.
+:TransportInformationEntity a bamm:Entity;
+    bamm:properties (:unNumber :shippingName :dgClass :dgPack);
+    bamm:preferredName "Transport information entity"@en;
+    bamm:description "Entity encapsulating the transport information data."@en.
+:RegulatoryInformationEntity a bamm:Entity;
+    bamm:properties (:limitations);
+    bamm:preferredName "regulatory information entity"@en;
+    bamm:description "Entity encapsulating the regulatory information."@en.
+:CarbonFootprintEntity a bamm:Entity;
+    bamm:properties (:co2FootprintTotal :co2Calculation :co2Standard);
+    bamm:preferredName "carbon footprint entity"@en;
+    bamm:description "Entity encapsulating the carbon footrprint."@en.
+:declarationOfConformity a bamm:Property;
+    bamm:preferredName "Declaration of Conformity"@en;
+    bamm:description "Declaration of conformity (CE) is describing a regulatory requirement from the EU."@en;
+    bamm:characteristic :LocateCharacteristic.
+:removalProcedure a bamm:Property;
+    bamm:preferredName "Removal procedure"@en;
+    bamm:description "The documents explain the removal and possible recycling options of the product."@en;
+    bamm:characteristic :LocateCharacteristic.
+:dueDiligence a bamm:Property;
+    bamm:preferredName "Due diligence"@en;
+    bamm:description "Report on Supply Chain Due Diligence policies. Link to document or website on its supply chain due diligence policies."@en;
```

Comment:
lower case for supply chain and due diligence

Reply:
done